realtime/utils.js

```diff
@@ -50,6 +50,17 @@ function isThisSubject(obj) {
 function isRoom(obj) {
   return obj.hasOwnProperty('type') && obj.hasOwnProperty('settings');
 }
+
+/**
+ * A function to see if an instance is an instance of a room
+ * Checks the name from the model
+ * @param  {Object}  obj - An object instance
+ * @returns {Boolean} - returns true if the name singular is room
+ */
+function isBotAction(obj) {
```

Comment:
still?

Reply:
We need this for the applyFilter in botEmit

Reply:
OK, but can you create and prioritize a separate user story to deal with this so we don't have to rely on the presence of attribute names to identify the model?

Reply:
Will do I created W-4481180 for this